src/custompresets.asm

```diff
@@ -0,0 +1,460 @@
+    LDA $7ED8CA : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8CC : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8CE : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8D0 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8D2 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8D4 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8D6 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8D8 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8DA : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8DC : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8DE : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8E0 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8E2 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8E4 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8E6 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8E8 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8EA : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8EC : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8EE : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8F0 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8F2 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8F4 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8F6 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8F8 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8FA : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8FC : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED8FE : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED900 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED902 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED904 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED906 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED908 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED90A : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED90C : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED90E : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED910 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED912 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED914 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED916 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED918 : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED91A : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED91C : STA $703000,X : INX #2 ;  Events, Items, Doors
+    LDA $7ED91E : STA $703000,X : INX #2 ;  Events, Items, Doors
+    RTL
+}
+
+custom_preset_load:
+{
+    LDA !sram_custom_preset_slot : ASL : TAX
+    LDA.l PresetSlot,X : TAX : LDY #$0000
+    INX #2 ; skip past "5AFE" word
```

Comment:
After this slot, the next slot would indicate how much space is actually used by the preset.  You had that at 1AEh but since we'd add two more bytes, we'd expect 1B0h.  Theoretically we could use that information to selectively load only some of the information, in case we add more information in the future to custom presets.  So in the future if we're storing 1C0h of info, then when loading an older custom preset with 1B0h info, we'd know to skip over the last 16 bytes of information.

Right now we'd probably just ignore it with an INX #2 when loading a custom preset.